README.md

```diff
@@ -118,11 +116,11 @@ See the image below for all configurable options on a response.
 **For Android Gradle Plugin 3.0**
 
 ```groovy 
-implementation 'com.nytimes.android:register:0.0.3'
+implementation 'com.nytimes.android:register:0.0.8'
 ```
 
 **For projects using older versions of the plugin**
 
 ```groovy
-compile 'com.nytimes.android:register:0.0.7'
+compile 'com.nytimes.android:register:0.0.8'
```

Comment:
These should also update to use the new `register-app` dependency as well. This dependency is only the library (which wraps BillingClient), and the `register-app` dependency is the companion app.

Reply:
👍